src/main/java/ch/njol/skript/events/EvtEntityShootBow.java

```diff
@@ -0,0 +1,92 @@
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.entity.EntityData;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.registrations.EventConverter;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.util.slot.Slot;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityShootBowEvent;
+import org.bukkit.inventory.EntityEquipment;
+import org.bukkit.inventory.EquipmentSlot;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import static ch.njol.skript.util.slot.EquipmentSlot.*;
+
+public class EvtEntityShootBow extends SkriptEvent {
+
+	static {
+		Skript.registerEvent("On Entity Shoot Bow", EvtEntityShootBow.class, EntityShootBowEvent.class,
```

Comment:
```suggestion
		Skript.registerEvent("Entity Shoot Bow", EvtEntityShootBow.class, EntityShootBowEvent.class,
```
Skript (I'm pretty sure) will add "On" for you.

Reply:
Skript will only add on into the pattern, not to the event name

https://github.com/SkriptLang/Skript/blob/e9c0f4d44f9bdec30f5b04e855c3ccba0d0a0af7/src/main/java/ch/njol/skript/doc/JSONGenerator.java#L140-L142
https://github.com/SkriptLang/Skript/blob/e9c0f4d44f9bdec30f5b04e855c3ccba0d0a0af7/src/main/java/ch/njol/skript/doc/HTMLGenerator.java#L574-L576

Reply:
its done in code:
<img width="965" alt="Screenshot 2025-03-16 at 4 33 03 PM" src="https://github.com/user-attachments/assets/4fbd896c-596e-4c92-a3d4-934575fc77d4" />

Reply:
Hmm well I sure as hell don't like that behavior, I'll get that fixed shortly then